precompile/allowlist/config_test.go

```diff
@@ -1,26 +1,27 @@
-// (c) 2019-2023, Ava Labs, Inc. All rights reserved.
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 
 package allowlist_test
```

Comment:
we can now move this to allowlisttest pkg

Reply:
this and config_test which also used to be `package allowlist_test`